src/xenia/kernel/objects/xthread.h

```diff
@@ -156,6 +160,8 @@ class XThread : public XObject {
   uint32_t pcr_address_;
   uint32_t thread_state_address_;
   cpu::ThreadState* thread_state_;
+  bool guest_thread_;  // Launched into guest code?
```

Comment:
Why the question? Is there a fuzzy line between host/guest threads? (like host created threads for guests ?)